messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -1183,9 +1198,15 @@ class MessagingTest : BaseMessagingTest() {
             replyToSenderId = replyToId?.let { toId }
         )
         assertFalse(senderStoredMsg.id.isNullOrBlank())
-        assertEquals(Model.StoredMessage.DeliveryStatus.SENDING, senderStoredMsg.status, testCase)
-        assertEquals(Model.MessageDirection.OUT, senderStoredMsg.direction, testCase)
-        assertEquals(fromId, senderStoredMsg.senderId, testCase)
+        if(fromId == toId){
+            assertEquals(Model.StoredMessage.DeliveryStatus.COMPLETELY_SENT, senderStoredMsg.status, testCase)
```

Comment:
in case that `fromId == toId`  the status of the message should be `COMPLETELY_SENT`